src/java/org/apache/cassandra/service/paxos/PaxosPrepare.java

```diff
@@ -1232,7 +1232,6 @@ public void serialize(Response response, DataOutputPlus out, int version) throws
         {
             if (response.isRejected())
             {
-                out.writeByte(0);
```

Comment:
Could we just move the contents of `serializeRejection` back here? I don't think it needs to be in a separate method, especially when we're serializing the success response inline. @dcapwell this bug only exists in the cep-15 branch